lib/purescript-cst/src/Language/PureScript/CST/Errors.hs

```diff
@@ -53,6 +53,7 @@ data ParserErrorType
   | ErrQualifiedName
   | ErrEmptyDo
   | ErrLexeme (Maybe String) [String]
+  | ErrQualifierLexeme Char
```

Comment:
By analogy with `ErrImportInDecl`, `ErrTypeInConstraint`, etc., should this be `ErrLexemeInQualified`?